superset-frontend/plugins/legacy-plugin-chart-sankey/src/ReactSankey.jsx

```diff
@@ -34,40 +34,42 @@ SankeyComponent.propTypes = {
 };
 
 export default styled(SankeyComponent)`
-  .superset-legacy-chart-sankey {
-    .node {
-      rect {
-        cursor: move;
-        fill-opacity: 0.9;
-        shape-rendering: crispEdges;
+  ${({ theme }) => `
+    .superset-legacy-chart-sankey {
+      .node {
+        rect {
+          cursor: move;
+          fill-opacity: 0.9;
```

Comment:
@michael-s-molina should we use the available opacities from the theme vars here?

Reply:
Nice catch! Done.